### ISSUE TITLE
Help menu: document role does not take any effect

### ISSUE THREAD

**Issue by jooyoungseo** [xability/maidr-legacy#68]
When help menu is displayed, "document" role must work as tagged to activate screen readers' virtual cursor (AKA browse mode). However, currently it does not take any effects. Usually, assigning tabindex 0 with "document" does the trick, but it does not work this time. Probably, it inherits from "dialog" ARIA role, but it was not the issue b/c removing the "dialog" did not make it work.

**Comment by ellvix** [comment]
@jooyoungseo is this still an issue? works for NVDA.

**Comment by jooyoungseo** [comment]
@ellvix  I think this issue still exists. Let's keep this open.
